pyproject.toml

```diff
@@ -59,6 +59,11 @@ forcefields = [
     "sevenn>=0.9.3",
     "torch<=2.2.1",       # incompatibility with dgl if newer versions are used
 ]
+classical_md = [
+    "mdanalysis>=2.7.0",
```

Comment:
Do these workflows actually require the latest version of mdanalysis etc? If so you can remove these pins from the `classical_md` section. You should keep them for the `strict` section however.

Reply:
I believe so. They actually require `2.8.0` for some features to work but that hasn't been released yet.

> If so you can remove these pins

A little confused by this, shouldn't I keep the pins if the dependency is required? Did you mean "If not" or am I missing something?